tests/test_graphql_api/test_reservation/test_query_permissions.py

```diff
@@ -0,0 +1,253 @@
+import datetime
+
+import pytest
+
+from reservations.choices import ReservationTypeChoice
+from tests.factories import ReservationFactory, ReservationUnitFactory, UnitFactory, UserFactory
+from tests.helpers import UserType
+
+from .helpers import reservations_query
+
+# Applied to all tests
+pytestmark = [
+    pytest.mark.django_db,
+    pytest.mark.usefixtures("_disable_elasticsearch"),
+]
+
+
+def test_reservation__query__regular_user_cannot_see_working_memo_for_other_users_reservation(graphql):
+    staff_user = UserFactory.create()
```

Comment:
In this test case, `staff_user` => `user`

Reply:
Fixed